### DISCUSSION TITLE
I cannot upload documents.

### DISCUSSION THREAD

**Discussion by okanissever** [community/community#107481] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/86436198/ebd38fcb-4fee-44d2-8a7c-b3d0720375c6)
There is no button or area here where I can upload a document. I can only take photos. What should I do ?

**Comment by ZephyrusBlaze** [comment]
Consider capturing the image directly instead of uploading it, as uploaded pictures are susceptible to modifications. Directly capturing the image will enable the GitHub Education team to verify its authenticity without concerns about potential edits.
